src/components/Button.js

```diff
@@ -2,21 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Calculator.css';
 
-const Button = ({ label }) => {
```

Comment:
| 🔴 Broken Links |
|-|
| ![image](https://github.com/CVILLA09/Math-Magicians/assets/93039359/e3602e22-15da-42a6-8e2e-e6804905c86e) |
| ![image](https://github.com/CVILLA09/Math-Magicians/assets/93039359/be5621c4-7f74-4c2d-9a58-4c6eddbe957d) |

- [ ] It would be great if you fix these broken links from the readme file to make your readme descriptive and professional 😊